src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPledgesRequest.cs

```diff
@@ -1,17 +1,28 @@
 ﻿using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace SFA.DAS.SharedOuterApi.InnerApi.Requests
 {
     public class GetPledgesRequest : IGetApiRequest
     {
-        public GetPledgesRequest(long? accountId = null)
+        public GetPledgesRequest(IEnumerable<string> sectors = null, long? accountId = null)
```

Comment:
I think these changes would benefit from refactoring - it's hard to read as it is.

I'd also be tempted to leave accountId as the first param, only because it's more commonly used and more important (and was here first).

Reply:
I have added similar NameValueCollection helper class here and formed querystring thru that.